code/game/objects/explosion.dm

```diff
@@ -7,6 +7,18 @@
 	else		return dy + (0.5*dx)
 
 
+
+/*
```

Comment:
If you want to add documentation do it the proper /tg/ way to ease porting later.

Reply:
Can you link me an example?

Reply:
Look it up on /tg/